taxi/views.py

```diff
@@ -57,7 +80,22 @@ class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
 class CarListView(LoginRequiredMixin, generic.ListView):
     model = Car
     paginate_by = 5
-    queryset = Car.objects.all().select_related("manufacturer")
+
+    def get_context_data(self, *, object_list=None, **kwargs):
```

Comment:
Provide return type
The same on other places

Reply:
add typehint
use typehint like: Optional[YOUR_VALUE_TYPE] = None